README.adoc

```diff
@@ -160,6 +160,10 @@ The executable of Jenkinsfile Runner allows its invocation with these CLI option
  -c (--cause) VAL            : A string describing the cause of the run.
                                It will be attached to the build so that it appears in the
                                build log and becomes available to plug-ins and pipeline steps.
+--scm FILE                   : A YAML file defining the SCM and optional credentials to use
```

Comment:
Would be nice to also reference the documentation page from somewhere in README. Can be done as a follow-up pull request